website/StandardAttributes.rst

```diff
@@ -0,0 +1,1388 @@
+         the image must be extrapolated.  This attribute tells the renderer
+         how to do this extrapolation.  The attribute contains either a pair
+         of comma-separated keywords, to specify separate extrapolation modes
+         for the horizontal and vertical directions; or a single keyword, to
+         specify extrapolation in both directions (e.g. "clamp,periodic" or
+         "clamp").  Extra white space surrounding the keywords is allowed,
+         but should be ignored by the renderer ("clamp, black " is equivalent
+         to "clamp,black").  The keywords listed below are predefined; some
+         renderers may support additional extrapolation modes:
+       </p>
+
+       <ul>
+         <li><tt>black</tt> - pixels outside the zero-to-one range are black </li>
+
+         <li><tt>clamp</tt> - texture coordinates less than 0.0 and greater than 1.0 are clamped
+           to 0.0 and 1.0 respectively. </li>
+
+         <li><tt>periodic</tt> - the texture image repeats periodically </li>
+
+         <li><tt>mirror</tt> - the texture image repeats periodically, but every other instance
+           is mirrored </li>
+       </li>
+     </td>
+   </tr>
+
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> multiView </b> </tt>
+     </td>
+     <td style="vertical-align: top; width:100px"> <tt> StringVector </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         Defines the view names for multi-view image files.  A multi-view
+         image contains two or more views of the same scene, as seen from
+         different viewpoints, for example a left-eye and a right-eye view
+         for stereo displays.  The <tt>multiView</tt> attribute lists the names of the
+         views in an image, and a naming convention identifies the channels
+         that belong to each view.
+       </p>
+     </td>
+   </tr>
+
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> deepImageState </b>
+     </tt> </td>
+     <td style="vertical-align: top; width:100px"> <tt> DeepImageState </tt>
+     </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         Specifies whether the pixels in a deep image are sorted and
+         non-overlapping.
+       </p>
+
+       <p style="padding-bottom:15px">
+         Note: this attribute can be set by application code that writes a
+         file in order to tell applications that read the file whether the
+         pixel data must be cleaned up prior to image processing operations
+         such as flattening.  The OpenEXR library does not verify that the
+         attribute is consistent with the actual state of the pixels.
+         Application software may assume that the attribute is valid, as long
+         as the software will not crash or lock up if any pixels are
+         inconsistent with the <tt>deepImageState</tt> attribute.
```

Comment:
Is it possible to link to InterpretingDeepPixels here? That defines the state of the library and describes how to convert image states

Reply:
Formatting of sphinx-based tables is really limited, and I couldn't get a layout I was happy with, so I switched to embedded html. Unfortunately, I'm not sure how to embed a link to the sphinx-generated content from inside the raw html. I'll look into it later.